frontend/src/pages/learningCenter/LearningCenter.tsx

```diff
@@ -49,7 +60,18 @@ const LearningCenterInner: React.FC<LearningCenterInnerProps> = React.memo(
     totalCount,
     favorites,
     updateFavorite,
+    viewType,
+    updateViewType,
   }) => {
+    const [sizeClass, setSizeClass] = React.useState<string>('m-ods-size-lg');
+    const { observe } = useDimensions({
+      breakpoints: { sm: 0, md: 600, lg: 750 },
```

Comment:
Is it better to use a react dependency for breakpoints rather than css breakpoints?

Reply:
We are basing it on the size of the component not the window. I did this because the nav bar can be shown/hidden, and the quickstart side panel can be shown/hidden.

Reply:
> 1. I don't have the US for this feature, but when a ListItem is starred and the list is sorted, the item stays in place, but if you sort again it stays on top of the list, I don't know if it will work better to re-render the list sorting when an element is sorted.
Talked thru this with @kywalker-rh, we came to the conclusion it is better not to jump the card when the favorite is toggled as it's a bit jarring even if we make the card visible afterwards (by scrolling the view if necessary)

Reply:
Ok perfect thank you very much!

Great work, LGTM!! 👌